src/MEDS_DEV/tasks/criteria/abnormal_lab/in_icu/blood_chemistry/elevated_creatinine/first_24h.yaml

```diff
@@ -0,0 +1,48 @@
+description: >-
+  This task predicts whether the patient will have elevated _creatinine_ values within the next 24
+  hours, given the first 24 hours of their admission to the ICU. To be included, patients must have
+  at least one measurement for _creatinine_ within the next 24 hours after their admission to ICU,
+  and their length of stay in ICU should be at least 24 hours.
+
+predicates:
+  icu_admission: ???
+  icu_discharge: ???
+  hospital_discharge: ???
+  death: ???
+  discharge_or_death:
+    expr: or(icu_discharge, death, hospital_discharge)
+
+  # Plain predicates
+  creatinine:
+    code: ???
+  abnormally_high_creatinine:
+    code: ???
+    value_min: ???
```

Comment:
Is this reference range different for men vs. women, or children vs. adults? Do we need more qualifiers and/or different ranges for different patients here?  Tagging @Gallifantjack for the clinical questions here.

Reply:
All the reference ranges were based on @mandiehyewon's suggestions, and only "Anemia" prediction task based on hemoglobin has a different range for men vs women. I'm also not sure if ACES does have the capacity to define predicates conditioned on other information (e.g., age, gender, pregnancy, etc).

Reply:
Good point about this @mmcdermott. While we can define cutoffs for elevated SCr, these values can be somewhat arbitrary and may vary between institutions. It's important to refer to established literature for guidance. According to [this source](https://www.ncbi.nlm.nih.gov/books/NBK544228/), the normal SCr levels for individuals with normal kidney function are:

- Men: 0.6 to 1.2 mg/dL
- Women: 0.5 to 1.1 mg/dL

The source also provides approximate measures of glomerular filtration rate (GFR) based on creatinine levels:

- Creatinine 1 mg/dL: Baseline for a patient with normal GFR
- Creatinine 2 mg/dL: Indicates about a 50% reduction in GFR
- Creatinine 4 mg/dL: Indicates about a 70% to 85% reduction in GFR
- Creatinine 8 mg/dL: Indicates about a 90% to 95% reduction in GFR

From this information, we can define multiple cutoffs for elevated creatinine levels. For instance, we might use 1.3 mg/dL—the higher end of the normal range for both men and women—as a threshold. The choice of cutoff depends on the specific condition we aim to detect:

- Critical Conditions (e.g., Creatinine ≥ 2.0 mg/dL): Levels that may require immediate medical intervention due to significant kidney function reduction.

- Mild Elevations (e.g., Creatinine ≥ 1.3 mg/dL): Levels that warrant further evaluation but may not indicate critical kidney failure.

Ultimately, the selected cutoff should align with the clinical context and the goals of the evaluation. Do we want to identify patients needing urgent care, or are we focusing on early detection of potential kidney issues that require monitoring?

Reply:
This is example of patient with mild SCr increase (with the cutoff 1.3 mg/dL) https://pmc.ncbi.nlm.nih.gov/articles/PMC4690708/#tfn3-permj08_1p0051

Reply:
Thanks @mandiehyewon -- this is super helpful. @justin13601, what is the status of our ability to define different reference ranges for different patients based on, e.g., static data? I think it is currently not possible; is that right?

Reply:
I think currently it is unsupported, but possibly doable by allowing users to create derived predicates joining normal plain predicates (ie. reference ranges) and a static predicates (ie. sex, which are also plain predicates under the hood)? Perhaps you would specify both `male` and `female` in `patient_demographics` so both are included, but then join it with different plain predicates - not sure if this works but I can look into it if this is of priority.

Reply:
@justin13601, can users currently create derived predicates with normal plain and static? Or would we need to add that? It seems passably simple to do so, if it isn't supported already.

Reply:
Nope, you're right. Can you file two github issues on ACES: 1) to disable the ability to create such derived predicates until it works properly and 2) to actually make it work properly and link them here? then if we can implement them in ACES fast enough we can use them for any labs where it is appropriate here.